data.md

```diff
@@ -512,10 +512,8 @@ However, `ByteMap` is just a wrapper around regular `Map`s.
 ```k
     syntax ByteMap ::= #setRange(ByteMap, Int, Int, Int) [function, functional, smtlib(setRange)]
```

Comment:
This function wasn't total before, due to the cast to `Int` in `#set`, but now the preconditions are tightening even more. Is it still necessary? Or do the new side conditions suffice to give the prover a good enough definedness condition?

Reply:
I've removed the note about this making `#set` more total, because it's not true.